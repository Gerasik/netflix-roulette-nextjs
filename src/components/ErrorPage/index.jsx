import React from 'react';
import styles from './styles.module.scss';

const ErrorPage = () => {
  return (
    <h1 className={styles.error}>
      <span className={styles.code}>404</span>
      Page not found
    </h1>
  );
};

export default ErrorPage;
