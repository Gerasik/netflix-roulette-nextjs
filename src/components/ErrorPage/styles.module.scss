.error {
  color: $fontColor;
  text-align: center;
  .code {
    color: $secondaryFontColor;
    margin-right: 0.5em;
  }
}
