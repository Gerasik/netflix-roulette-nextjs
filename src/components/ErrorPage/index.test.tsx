import React from 'react';
import { shallow } from 'enzyme';

import ErrorPage from '.';

describe('<ErrorPage />', () => {
  it('should to be match snapshot  ', () => {
    const wrapper = shallow(<ErrorPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
