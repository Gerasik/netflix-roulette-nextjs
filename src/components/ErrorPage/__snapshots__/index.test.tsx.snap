// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage /> should to be match snapshot   1`] = `
<h1
  className="error"
>
  <span
    className="code"
  >
    404
  </span>
  Page not found
</h1>
`;
